ovs_events: build failure on OSX

```
src/utils_ovs.c:1097:5: error: 'sem_init' is deprecated [-Werror,-Wdeprecated-declarations]
    sem_init(&new_cb->result.sync, 0, 0);
    ^
/usr/include/sys/semaphore.h:55:5: note: 'sem_init' has been explicitly marked deprecated here
int sem_init(sem_t *, int, unsigned int) __deprecated;
    ^
mv -f src/.deps/network_la-network.Tpo src/.deps/network_la-network.Plo
src/utils_ovs.c:1111:11: error: implicit declaration of function 'sem_timedwait' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
      if (sem_timedwait(&new_cb->result.sync, &ts) < 0) {
          ^
src/utils_ovs.c:1111:11: note: did you mean 'sem_trywait'?
/usr/include/sys/semaphore.h:58:5: note: 'sem_trywait' declared here
int sem_trywait(sem_t *);
    ^
src/utils_ovs.c:1125:5: error: 'sem_destroy' is deprecated [-Werror,-Wdeprecated-declarations]
    sem_destroy(&new_cb->result.sync);
    ^
/usr/include/sys/semaphore.h:53:5: note: 'sem_destroy' has been explicitly marked deprecated here
int sem_destroy(sem_t *) __deprecated;
    ^
depbase=`echo src/powerdns.lo | sed 's|[^/]*$|.deps/&|;s|\.lo$||'`;\
	/bin/sh ./libtool  --tag=CC   --mode=compile gcc -DHAVE_CONFIG_H -I. -I./src  -I./src -I./src/daemon -DPREFIX='"/opt/collectd"' -DCONFIGFILE='"/opt/collectd/etc/collectd.conf"' -DLOCALSTATEDIR='"/opt/collectd/var"' -DPKGLOCALSTATEDIR='"/opt/collectd/var/lib/collectd"' -DPLUGINDIR='"/opt/collectd/lib/collectd"' -DPKGDATADIR='"/opt/collectd/share/collectd"'  -Wall -Werror -g -O2 -MT src/powerdns.lo -MD -MP -MF $depbase.Tpo -c -o src/powerdns.lo src/powerdns.c &&\
	mv -f $depbase.Tpo $depbase.Plo
3 errors generated.
make[1]: *** [src/ovs_events_la-utils_ovs.lo] Error 1
make[1]: *** Waiting for unfinished jobs....
libtool: compile:  gcc -DHAVE_CONFIG_H -I. -I./src -I./src -I./src/daemon -DPREFIX=\"/opt/collectd\" -DCONFIGFILE=\"/opt/collectd/etc/collectd.conf\" -DLOCALSTATEDIR=\"/opt/collectd/var\" -DPKGLOCALSTATEDIR=\"/opt/collectd/var/lib/collectd\" -DPLUGINDIR=\"/opt/collectd/lib/collectd\" -DPKGDATADIR=\"/opt/collectd/share/collectd\" -Wall -Werror -g -O2 -MT src/ovs_stats_la-ovs_stats.lo -MD -MP -MF src/.deps/ovs_stats_la-ovs_stats.Tpo -c src/ovs_stats.c  -fno-common -DPIC -o src/.libs/ovs_stats_la-ovs_stats.o
libtool: compile:  gcc -DHAVE_CONFIG_H -I. -I./src -I./src -I./src/daemon -DPREFIX=\"/opt/collectd\" -DCONFIGFILE=\"/opt/collectd/etc/collectd.conf\" -DLOCALSTATEDIR=\"/opt/collectd/var\" -DPKGLOCALSTATEDIR=\"/opt/collectd/var/lib/collectd\" -DPLUGINDIR=\"/opt/collectd/lib/collectd\" -DPKGDATADIR=\"/opt/collectd/share/collectd\" -Wall -Werror -g -O2 -MT src/ovs_stats_la-utils_ovs.lo -MD -MP -MF src/.deps/ovs_stats_la-utils_ovs.Tpo -c src/utils_ovs.c  -fno-common -DPIC -o src/.libs/ovs_stats_la-utils_ovs.o
mv -f src/.deps/ovs_events_la-ovs_events.Tpo src/.deps/ovs_events_la-ovs_events.Plo
src/utils_ovs.c:1097:5: error: 'sem_init' is deprecated [-Werror,-Wdeprecated-declarations]
    sem_init(&new_cb->result.sync, 0, 0);
    ^
/usr/include/sys/semaphore.h:55:5: note: 'sem_init' has been explicitly marked deprecated here
int sem_init(sem_t *, int, unsigned int) __deprecated;
    ^
src/utils_ovs.c:1111:11: error: implicit declaration of function 'sem_timedwait' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
      if (sem_timedwait(&new_cb->result.sync, &ts) < 0) {
          ^
src/utils_ovs.c:1111:11: note: did you mean 'sem_trywait'?
/usr/include/sys/semaphore.h:58:5: note: 'sem_trywait' declared here
int sem_trywait(sem_t *);
    ^
src/utils_ovs.c:1125:5: error: 'sem_destroy' is deprecated [-Werror,-Wdeprecated-declarations]
    sem_destroy(&new_cb->result.sync);
    ^
/usr/include/sys/semaphore.h:53:5: note: 'sem_destroy' has been explicitly marked deprecated here
int sem_destroy(sem_t *) __deprecated;
    ^
3 errors generated.
make[1]: *** [src/ovs_stats_la-utils_ovs.lo] Error 1
```

--- a/configure.ac
+++ b/configure.ac
@@ -6005,6 +6005,11 @@ if test "x$ac_system" = "xLinux"; then
   if test "x$c_cv_have_clock_boottime_monotonic" = "xyes"; then
     plugin_cpusleep="yes"
   fi
+
+  if test "x$with_libyajl" = "xyes" && test "x$with_libyajl2" = "xyes"; then
+    plugin_ovs_events="yes"
+    plugin_ovs_stats="yes"
+  fi
 fi
 
 if test "x$ac_system" = "xOpenBSD"; then
@@ -6179,11 +6184,6 @@ fi
 
 if test "x$with_libyajl" = "xyes"; then
   plugin_log_logstash="yes"
-fi
-
-if test "x$with_libyajl" = "xyes" && test "x$with_libyajl2" = "xyes"; then
-  plugin_ovs_events="yes"
-  plugin_ovs_stats="yes"
 fi
 
 if test "x$with_libperl" = "xyes" && test "x$c_cv_have_perl_ithreads" = "xyes"; then

